### ISSUE TITLE
Suggestion: Transport Rings Emit Redstone Signal

### ISSUE THREAD

**Issue by ShicKla** [MrJake222/AUNIS#192]
Would it be possible to have the Transport Rings block emit a redstone signal when they are activated?

**Comment by MrJake222** [comment]
Yes, it would. And I also need to look into the DHD's redstone code because I found a bug where it won't stop emitting the signal :thinking:
